brush-core/src/error.rs

```diff
@@ -51,7 +51,8 @@ pub enum Error {
     #[error("not yet implemented: {0}")]
     Unimplemented(&'static str),
 
-    /// The requested functionality has not yet been implemented in this shell; it is tracked in a GitHub issue.
+    /// The requested functionality has not yet been implemented in this shell; it is tracked in a
```

Comment:
It looks like a number of the changes in the PR are reformatting; are they intentional? And if so, would you please split them off into a separate PR to keep them separated from the more intentional code changes?

Reply:
Yeah, I did run the codebase through the formatter. I was going to make a separate PR for that as well, but didn't want it to seem like I was PR spamming :sweat_smile:. I'll go ahead and get that going though.